### PR TITLE
feat(APP-3648): Improve Explore page hero section + layout

### PR DESCRIPTION
**Pull request by thekidnamedkd**
## Description

Update background text, typeface, layout, carousel content, etc

Task: [APP-3648](https://aragonassociation.atlassian.net/browse/APP-3648)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.

[APP-3648]: https://aragonassociation.atlassian.net/browse/APP-3648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,10 @@
     <meta name="app-version" content="%VITE_REACT_APP_DEPLOY_VERSION%" />
 
     <link href="./styles.css" rel="stylesheet" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Sora:wght@400&display=swap"
+      rel="stylesheet"
+    />
```
Comment (cgero-eth) [inline index.html:33]: We can delete the `syne` font load if not used else where

File: src/assets/locales/en/common.json [hunk 2]
```diff
@@ -1010,8 +1010,8 @@
   },
   "explore": {
     "hero": {
-      "title": "Explore the DAO World",
-      "subtitle1": "Build your DAO, explore communities, and find inspiration for your project all in one place."
+      "title": "Governed on Aragon",
```
Comment (cgero-eth) [inline src/assets/locales/en/common.json:1013]: Here as well, we should update the translations on Crowdin if not already updated

File: src/components/ctaCard/index.tsx [hunk 3]
```diff
@@ -27,10 +28,10 @@ const CTACard: React.FC<Props> = props => {
       </Content>
 
       <Button
-        variant={props.actionAvailable ? 'primary' : 'tertiary'}
-        size="lg"
+        variant={props.isPrimary ? 'primary' : 'secondary'}
+        size="md"
         disabled={!props.actionAvailable}
-        onClick={() => props.onClick(props.path)}
+        href={props.path}
```
Comment (cgero-eth) [inline src/components/ctaCard/index.tsx:34]: Some of the actions should open the link on a new tab

File: src/containers/hero/index.tsx [hunk 4]
```diff
@@ -1,90 +1,54 @@
...
+import Blueprint from 'assets/images/blueprintTexture.svg';
 import {useTranslation} from 'react-i18next';
 import {GridLayout} from 'components/layout';
 
 function Hero() {
   const {t} = useTranslation();
+
   return (
     <Container>
+      <StyledImage src={Blueprint} alt="Background Texture" />
```
Comment (cgero-eth) [inline src/containers/hero/index.tsx:12]: On the design, it looks like we have a solid background color on the header. Current implementation displays the grid on the header as well instead. 
Design:
Screenshot 2024-10-28 at 16 22 31
Implementation:
Screenshot 2024-10-28 at 16 23 25

File: src/containers/carousel/index.tsx [hunk 5]
```diff
@@ -112,8 +112,7 @@ const Carousel: React.FC = () => {
 };
 
 const DesktopCTA = styled.div.attrs({
-  className:
```
Comment (cgero-eth) [inline src/containers/carousel/index.tsx]: I remember Selim suggested to remove the slider, could you double check with him if the slider should be removed or if we should keep it?
Screenshot 2024-10-28 at 16 24 16

**Review by cgero-eth — COMMENTED** [review]
Thanks for the updates @thekidnamedkd!

File: index.html (hunk 1 above)
Comment (thekidnamedkd) [reply index.html:33]: I thought this too maybe, but it is used on the notFound.ts 404 still. Verifying with Selim.

Comment (thekidnamedkd) [reply index.html:33]: Selim says to keep for 404.

**Review by cgero-eth — APPROVED** [review]
LGTM, thanks @thekidnamedkd!

**Pull request merged**